src/power_grid_model/validation/assertions.py

```diff
@@ -84,8 +84,8 @@ def assert_valid_batch_data(
         symmetric: A boolean to state whether input data will be used for a symmetric or asymmetric calculation
 
     Raises:
-        KeyError, TypeError or ValueError if the data structure is invalid.
-        ValidationException if the contents are invalid.
+        Error: KeyError, TypeError or ValueError if the data structure is invalid.
+          ValidationException if the contents are invalid.
```

Comment:
these should have the same indentation levels, right?

if not, please make these bullet points

Reply:
The indentation of the long description is 2 spaces.

Reply:
yes but in that case, i think the Error on should also be indented. The two different options are KeyError, TypeEroor or ValueError and ValidationException